templates/server/src/main/resources/archetype-resources/core/src/test/resources/db/changelog/changelog-master.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>   

```

Comment:
we have the same file in `src/main/resources` and `src/test/resources`. This IMHO does not make sense.
Like with flyway we should use `db/test` instead of `db/changelog` for tests.
Will it be best practice to use a completely redundant/isolated db config for liquibase for testing vs. main?

Reply:
I have configured this for liquibase separately in Groovy script.